examples/firewall/go.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
Instead of adding a new `go.sh` script create a symlink to an existing `examples/go.sh` script. 

This should do it:
`ln -s ../go.sh go.sh`